components/Projects.js

```diff
@@ -19,11 +24,25 @@ export default function Projects({
       }
     }
     const newId = testEntry(lastEntry);
-    setEditorContent({title: "", text: "", id: newId});
+    setEditorContent({title: "", text: "", id: newId, wordcount: 0});
     handleChangePage("editor");
   }
+  function handleDeletion(entryYouWantDeleted) {
+    setIdForDeletion(entryYouWantDeleted);
+    setDeletePopup(true);
+  }
   return (
     <AllFiles>
+      {deletePopup === true ? (
+        <DeleteConfirm
+          setDeletePopup={setDeletePopup}
+          idForDeletion={idForDeletion}
+          myProjects={myProjects}
+          setMyProjects={setMyProjects}
+        />
+      ) : (
+        <></>
+      )}
```

Comment:
You could also write it like this:

```suggestion
      {deletePopup && (
        <DeleteConfirm
          setDeletePopup={setDeletePopup}
          idForDeletion={idForDeletion}
          myProjects={myProjects}
          setMyProjects={setMyProjects}
        />
      )}
```

Reply:
Implementing the code changes produces a prettier error